internal/support/notifications/rest_notification.go

```diff
@@ -53,6 +53,13 @@ func notificationHandler(w http.ResponseWriter, r *http.Request) {
 
 		if n.Severity == models.NotificationsSeverity(models.Critical) {
 			LoggingClient.Info("Critical severity scheduler is triggered for: " + n.Slug)
+			n, err = dbClient.GetNotificationById(n.ID)
+			if err != nil {
+				http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
- I was debating over the choice made here for indicating which HTTP error it was, but agree that this is the best choice possible: `http.StatusInternalServerError` 👍

Reply:
In this line, since the record has been persisted successfully, it would be some unknown error.  That's why I choose StatusInternalServerError

Reply:
Yep, I'm good with that 👍